package.json

```diff
@@ -33,17 +33,14 @@
     "start": "next start",
     "test": "vitest run",
     "test:watch": "vitest",
-    "test:playwright": "npx playwright",
```

Comment:
There were a lot of scripts previously. We really only need two. If you use Playwright for the first time, it will prompt you to install, so it seemed unnecessary to add a script for it. Same for the E2E report. If we need it, we can run the command.